airflow/opt/providers/etna/etna/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 
```

Comment:
Exporting the most useful pieces of this into the top level module to make imports easy:

```
from etna import dag, rollup_dag, etc
```